src/test/java/com/exasol/projectkeeper/validators/dependencies/DependenciesValidatorIT.java

```diff
@@ -40,6 +41,17 @@ void testWrongContent() throws IOException, VerificationException {
         assertThat(output, containsString("[ERROR] E-PK-53: The dependencies.md file has outdated content."));
     }
 
+    @Test
+    void testWrongContentWithNonDefaultRepository() throws IOException, VerificationException {
```

Comment:
Can you also test that the link was extracted and written to the dependencies file from a non-default repository?

Reply:
the difference between non-default and default is only in the crawler module. Unfortunately, I can't unit test it, since it depends on injected maven dependencies. As long as the repository is discovered the is no different path to cover.
(Integration tests are time expensive here...)